js/private/js_helpers.bzl

```diff
@@ -138,6 +138,43 @@ def gather_npm_package_store_deps(targets):
 
     return depset([], transitive = npm_package_store_deps)
 
+def copy_js_file_to_bin_action(ctx, file):
+    if ctx.label.workspace_name != file.owner.workspace_name or ctx.label.package != file.owner.package:
+        msg = """
+
+Expected to find source file {file_basename} in {this_package}, but instead it is in {file_package}.
```

Comment:
Hmm. Should probably quote the packages since the root package is the empty string `""`

Reply:
A few spots below as well